q2_dada2/plugin_setup.py

```diff
@@ -468,7 +468,6 @@
                 'https://github.com/benjjneb/LRASManuscript'
 )
 
```

Comment:
```suggestion


```

Reply:
Let's leave this as-is (with the extra newline in there) so this isn't included in the PR changes (since no functionality has changed within `plugin_setup.py` as a result of this refactor).